web/frontend/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "type": "module",
   "engines": {
-    "node": ">= 12.16"
+    "node": ">=14.13.1 <17.0.0"
```

Comment:
#6 removes these engine specifications so we may want to either remove them totally or move this to the root package